ov/config.go

```diff
@@ -0,0 +1,13 @@
+package ov
+
+import (
+	_ "github.com/HewlettPackard/oneview-golang/utils"
+)
+
+type Configuration struct {
+	UserName   string `json:"username"`
```

Comment:
IfMatch and SSL verify parameters are missing. It would be good if we give all the parameters

Reply:
Added